src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import { Outlet } from 'react-router-dom';
```

Comment:
[OPTIONAL] I think we don't need to use this component from react-router-dom. This one is made to create nested UI components which we wont be using in this project.

Reply:
Oh ok.
I'm actually using the v6 one and I thought Outlet allows for switching pages just as the Switch did in v5.
I will remove it and see. 
Thanks bro 👍